File: ComplementApp.API/Dtos/ClavePresupuestalContableDto.cs
using ComplementApp.API.Models;

namespace ComplementApp.API.Dtos
{
    public class ClavePresupuestalContableDto
    {
        public int ClavePresupuestalContableId { get; set; }
        public int DocumentoCompromisoId { get; set; }
        public long Crp { get; set; }
        public string Pci { get; set; }
        public string Dependencia { get; set; }
        public string DependenciaDescripcion { get; set; }
        public string Detalle4 { get; set; }
        public ValorSeleccion RubroPresupuestal { get; set; }
        public ValorSeleccion Tercero { get; set; }
        public ValorSeleccion SituacionFondo { get; set; }
        public ValorSeleccion FuenteFinanciacion { get; set; }
        public ValorSeleccion RecursoPresupuestal { get; set; }
        public ValorSeleccion UsoPresupuestal { get; set; }
        public ValorSeleccion RelacionContable { get; set; }
        public RelacionContable RelacionContableDto { get; set; }
        public ValorSeleccion CuentaContable { get; set; }
        public int EstadoModificacion { get; set; }

    }
}

File: ComplementApp-SPA/src/app/_dto/clavePresupuestalContableDto.ts
import { ValorSeleccion } from './valorSeleccion';

export interface ClavePresupuestalContableDto {
  clavePresupuestalContableId: number;
  cdpId: number;
  crp: number;
  pci: string;
  dependencia: string;
  dependenciaDescripcion: string;
  rubroPresupuestal: ValorSeleccion;
  tercero: ValorSeleccion;
  situacionFondo: ValorSeleccion;
  fuenteFinanciacion: ValorSeleccion;
  recursoPresupuestal: ValorSeleccion;
  usoPresupuestal: ValorSeleccion;
  relacionContable: ValorSeleccion;
  estadoModificacion: number;
}
